recipe/build_osx_arm64.sh

```diff
@@ -16,6 +16,12 @@ export CMAKE_PREFIX_PATH=${PREFIX}
 # Filter CMAKE_PREFIX_PATH out of CMAKE_ARGS because scikit-build needs to set it
 export SKBUILD_CONFIGURE_OPTIONS=$(echo $CMAKE_ARGS | sed -e 's/\-DCMAKE_INSTALL_PREFIX\=[^ ]* //g')
 
+# 10.14 required for some C++17 features used in TBB
+MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
+  - "10.14"                # [osx and x86_64]
+MACOSX_SDK_VERSION:        # [osx and x86_64]
+  - "10.14"                # [osx and x86_64]
```

Comment:
This isn't shell script. Does this go in `recipe/conda_build_config.yml`?

https://conda-forge.org/docs/maintainer/knowledge_base.html#requiring-newer-macos-sdks

Reply:
that's my fault, I'll fix it